pay-api/src/pay_api/dtos/eft_shortname.py

```diff
@@ -71,10 +71,14 @@ class EFTShortNameRefundGetRequest:
     """EFT Short name refund DTO."""
 
     statuses: List[str]
+    short_name_id: int = None
 
     @classmethod
     def from_dict(cls, data: dict):
         """Convert from request json to EFTShortNameRefundDTO."""
-        input_string = data.get("status", "")
+        input_string = data.get("statuses", "")
+        short_name_id = None
+        if data.get("shortNameId", None) is not None:
```

Comment:
please use serialization instead of doing this

Reply:
we eventually want to get rid of dto.state = dto.state.split(",") etc somehow in the converter

Reply:
I'll see if we can put in:

typical register structure unstructure hook - new type DelimitedString